src/containers/assignees/reducer.js

```diff
@@ -0,0 +1,61 @@
+import {
+  ADD_ASSIGNEE_TO_CARD, ADD_RESPONSIBLE_TO_CARD, GET_ALL_ASSIGNEES_IN_CARD, GET_RESPONSIBLE_FOR_CARD,
+  REMOVE_ASSIGNEE_FROM_CARD, REMOVE_RESPONSIBLE_FROM_CARD } from './constants'
+
+const initialState = {
+  assignees: [],
```

Comment:
Prefer to use the term `data` to have when you call it `assignees.data` and not `assignees.assignees`.